### ISSUE TITLE
Proposal logo/Icon

### ISSUE THREAD

**Issue by amirdesaingrafis** [T-Rex96/Easy_xkcd#132]
hi, i want to contribute about this great project. I'm not saying your logo is ugly. if you allow me to propose a new logo / icon for you. thank you

**Comment by TacoTheDank** [comment]
go ahead, as long as you made it yourself, and not stolen from flaticons like ive already exposed two other people doing and got them banned from utopian.io :)

**Comment by amirdesaingrafis** [comment]
of course, I will design a new logo without changing the old logo construction, just a little improvement to look more tidy. hope you like it.😄

**Comment by TacoTheDank** [comment]
@amirdesaingrafis sorry if me saying that seemed a bit confronting lol, im just tired of having to deal with people who just steal icons from websites (and say they made them themselves, which really drives me up walls) to try to "contribute" to open source projects for money (not to mention potential legal problems, as the icons are licensed), and then lash out at me when i call them out for it. usually they also have shitty grammar as well, so its pretty easy to tell who's fake and who isn't
